docs/data-management/post-session.md

```diff
@@ -286,8 +286,23 @@ To support backward compatibility (and some extra, currently unsupported feature
     ```
 
 ### Convert eye-tracking into BIDS with *bidsphysio*
+- [ ] Pull the latest docker image of esavary/bidsphysio with:
+``` shell
+docker pull esavary/bidsphysio
+```
+- [ ] Open [schedule_ET_data.tsv](../code/physioconv/schedule_ET_data.tsv) to find the phase encoding direction and the name of the .EDF file corresponding to the session you want to convert.
+- [ ] In the folder containing your data, create a new folder named metadata. Copy the file containing information about the eye-tracker, named [info_ET.json](../code/physioconv/info_ET.json), inside this metadata folder.
+- [ ]  Run the following command for the file corresponding to the DWI (Diffusion-Weighted Imaging) data, replacing PATH_RAW_DATA,PATH_OUTPUT, DWI_FILENAME, SESSION_NUMBER, and PHASE_ENCODING_DIRECTION with the information corresponding to the file you want to process:
+``` shell
+docker run -u $( id -u ):$( id -g ) --rm -it -v PATH_RAW_DATA:/data -v PATH_OUTPUT:/output --entrypoint=/opt/venv/bin/python esavary/bidsphysio /opt/venv/bin/edf2bidsphysio --infile DWI_FILENAME --bidsprefix /output/dwi/sub-001_ses-SESSION_NUMBER_acq-highres_dir-PHASE_ENCODING_DIRECTION -m /data/metadata/info_ET.json
+
+```
+- [ ] Run the following command for the three files corresponding to the functional tasks:
+``` shell
+docker run -u $( id -u ):$( id -g ) --rm -it -v PATH_RAW_DATA:/data -v PATH_OUTPUT:/output --entrypoint=/opt/venv/bin/python bidsphysio /opt/venv/bin/edf2bidsphysio --infile /data/FILENAME --bidsprefix /output/session01/func/sub-001_ses-SESSION_NUMBER_task-TASK_NAME_dir-PHASE_ENCODING_DIRECTION -m /data/metadata/info_ET.json
+```
```

Comment:
```suggestion
- [ ] Run the following command for the three files corresponding to the functional tasks:
    ``` shell
    docker run -u $( id -u ):$( id -g ) --rm -it \
        -v PATH_RAW_DATA:/data -v PATH_OUTPUT:/output \
        --entrypoint=/opt/venv/bin/python bidsphysio \
        /opt/venv/bin/edf2bidsphysio \
        --infile /data/FILENAME \
        --bidsprefix /output/session01/func/sub-001_ses-SESSION_NUMBER_task-TASK_NAME_dir-PHASE_ENCODING_DIRECTION -m /data/metadata/info_ET.json
    ```
```